### DISCUSSION TITLE
Previous versions (5.5) of UVR for Mac has dissapeared ?

### DISCUSSION THREAD

**Discussion by ZenWFWSmith** [Anjok07/ultimatevocalremovergui#823] [Q&A]
Hello,

I'm new to the site here, so sorry if I'm doing something wrong :) 
I've downloaded a version of UVR on Sunday (I'm on Catalina 10.15.7) and everything worked fine until this morning where I had the strange idea to download the 5.6 which is not compatible with my OS... and of course, I've scratched my previous version.
I'm searching through all previous releases and there's only windows (.exe) available... no .dmg's :( 
And when I try to do download the 5.5... it's the 5.6 which is downloaded :( 
Can someone help ? 
Many thanks

**Comment by ZenWFWSmith** [comment]
I found it ... sorry for bothering :p

**Comment by BadaBingCherry** [comment]
Been awhile since your query but, where did you find 5.5? Have a link?
